fearless/Modules/Staking/SelectValidatorsFlow/ValidatorSearch/ValidatorSearchViewController.swift

```diff
@@ -9,6 +9,12 @@ final class ValidatorSearchViewController: UIViewController, ViewHolder {
 
```

Comment:
I think we need to fix the input indicator of the search field. Currently it's blue, I think it should be white.